src/main/java/bio/terra/cli/app/utils/LocalProcessLauncher.java

```diff
@@ -50,7 +50,6 @@ public void launchProcess(
       Map<String, String> procEnvVars = procBuilder.environment();
       procEnvVars.putAll(envVars);
     }
-    procBuilder.inheritIO();
```

Comment:
Test output wasn't handled correctly for tests using local process runner. There are currently no tests using local process runner. (I was investigating using local process runner for Gradle Enterprise Test Distribution.)

Before this change:

- localProcessLauncher.streamOutputForProcess() wasn't doing anything. If you commented it out, tool output still appeared
- TestCommand sets up stdOut/stdErr for test. After running tool, stdOut did not contain tool output. Hence, a test wouldn't be able to assert anything about tool stdout (like stdout contains bq dataset id).

In debugger, I noticed that [process.getInputStream()](https://github.com/DataBiosphere/terra-cli/blob/main/src/main/java/bio/terra/cli/app/utils/LocalProcessLauncher.java#L64) was NullInputStream. Turns out this was because of [procBuilder.inheritIO()](https://github.com/DataBiosphere/terra-cli/blob/main/src/main/java/bio/terra/cli/app/utils/LocalProcessLauncher.java#L53). That [sets up some redirects](https://docs.oracle.com/javase/7/docs/api/java/lang/ProcessBuilder.html#inheritIO()), which[ caused NullInputStream to be returned](https://docs.oracle.com/javase/7/docs/api/java/lang/ProcessBuilder.html#inheritIO():~:text=case%2C%20Process.getOutputStream()%20will%20return%20a%20null%20output%20stream%2C).

After deleting procBuilder.inheritIO(), things worked properly:

- If I comment out localProcessLauncher.streamOutputForProcess(), I don't see tool output
- TestCommand stdOut contains tool output